### PR TITLE
Fix particle emitter rotation speed

### PR DESCRIPTION
**Pull request by D8H**

### CODE REVIEW

File: Extensions/ParticleSystem/particleemitterobject-pixi-renderer.ts [hunk 1]
```diff
@@ -119,7 +119,7 @@ namespace gdjs {
               },
               minStart: objectData.emitterForceMax,
```
Comment (ClementPasteau) [inline Extensions/ParticleSystem/particleemitterobject-pixi-renderer.ts:120]: btw, should this be `emitterForceMin` ?
We don't use this prop anymore

Comment (D8H) [reply Extensions/ParticleSystem/particleemitterobject-pixi-renderer.ts]: Indeed, it's just that the whole configuration values are overridden by the setters at the start so it didn't do a bug.

Comment (ClementPasteau) [reply Extensions/ParticleSystem/particleemitterobject-pixi-renderer.ts]: nice

File: Extensions/ParticleSystem/particleemitterobject-pixi-renderer.ts [hunk 2]
```diff
@@ -273,6 +277,18 @@ namespace gdjs {
       const behavior: any = this.emitter.getBehavior('moveAcceleration');
       behavior.accel.x = x;
       behavior.accel.y = y;
+      this._updateRotateFromSpeedFlag();
+    }
+
+    private _updateRotateFromSpeedFlag() {
```
Comment (ClementPasteau) [inline Extensions/ParticleSystem/particleemitterobject-pixi-renderer.ts:283]: maybe a small comment explaining why we need to do that?

**Review by ClementPasteau — APPROVED** [review]
lgtm

**Review by 4ian — APPROVED** [review]

**Pull request merged**